ci/management/commands/generate_reports.py

```diff
@@ -19,7 +20,7 @@
 
 def mk_validator(django_validator):
     def validator(inputstr):
-        django_validator(inputstr)
+        django_validator()(inputstr)
```

Comment:
The `django_validator` here is the class, it needs to be instantiated before it'll validate, this was a bug. After fixing this the various `hub` urls below in the fixtures started failing, you'll notice I've updated those to `hub.example.com` etc... to pass the validator (which is now working).